scapy/contrib/isotp.py

```diff
@@ -168,6 +172,97 @@ def defragment(can_frames, use_extended_addressing=None):
         return results[0]
 
 
+class ISOTPHeader(CAN):
+    name = 'ISOTPHeader'
+    fields_desc = [
+        FlagsField('flags', 0, 3, ['error', 'remote_transmission_request', 'extended']),  # noqa: E501
```

Comment:
Please don't introduce `noqa` comments but modify your code to make it fit on a single line.